presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7269,4 +7318,26 @@ private Session withTimestampPrecision(Session session, String precision)
                 .setCatalogSessionProperty(catalog, "timestamp_precision", precision)
                 .build();
     }
+
+    private static class TestSystemAccessControl
```

Comment:
Do we need this additional system access control at all? You have each tests doubled while I think the only one that is using `privilege()` is enough here. What am I missing?